src/apps/omis/transformers.js

```
@@ -0,0 +1,23 @@
+const { isArray } = require('lodash')
+
+function transformToOptions (items, labelIteratee = (item) => item.name) {
+  if (!isArray(items)) { return [] }
```

Should it return empty array if it's not an array to begin with? It could potentially swallow a user error.